Automattic/wp-calypso
# Remove Secure Your Brand Test

The experiment was declared unsuccessful

This is not a revert of #46078 and #47708. We keep the data layer because it's still in use for another experiment

#### Changes proposed in this Pull Request

Ends Secure Your Brand and removes code that's no longer in use

#### Testing instructions

1. Sign up with a new user and site from /start/user
2. Select a domain and a plan from the flow
3. Proceed to checkout without paying and remove the cart items
4. Assign yourself to the `test` group of `freePlansDomainUpsell`
5. Launch the site from /home using a free site and free domain
6. The Domain Upsell final step should be there and should work, selecting a paid domain should redirect to checkout
